cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -217,10 +217,16 @@ set(EMSCRIPTEN_SYSROOT "${_emcache_output}/sysroot")
 list(APPEND CMAKE_FIND_ROOT_PATH "${EMSCRIPTEN_SYSROOT}")
 list(APPEND CMAKE_SYSTEM_PREFIX_PATH /)
 
-if ($ENV{CFLAGS} MATCHES "MEMORY64")
+if (${CMAKE_C_FLAGS} MATCHES "MEMORY64")
```

Comment:
What sets this?

Reply:
https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_FLAGS.html  (its set from both the CFLAGS env var and also from the -DCMAKE_C_FLAGS command line flag).

Reply:
Who sets `MEMORY64` in the env, though?

Reply:
We are looking for the `-sMEMORY64` flag here.  I just use `MEMORY64` because it could be `-s MEMORY64` or `-sMEMORY64`.

The key I'm looking for doesn't change with this PR.. I'm just looking in a better place.

Reply:
I see now, thanks.